### ISSUE TITLE
Greek letters bug

### ISSUE THREAD

**Issue by Ethereal-Fox** [wxMaxima-developers/wxmaxima#1654]
There is a bug in wxMaxima 22.04.0.
"Change names of Greek letters to Greek letters" option doesn't work correctly for beta, theta, lambda.
But it seems it's Ok if I enter by pressing the ESC key and then starting to type the Greek character’s name.
Also, the letters in the first and second lines are displayed in different styles
![Screenshot](https://user-images.githubusercontent.com/103583938/163577033-c1b6cb54-ea40-438f-8fb6-54db57df61a1.png)
.

**Comment by daute** [comment]
I am not sure, if that is intended, but "lambda" is also a Maxima function (https://maxima.sourceforge.io/docs/manual/maxima_singlepage.html#index-lambda), if it would be automatically converted to the greek letter "λ", it would be impossible to enter this function.

**Comment by Ethereal-Fox** [comment]
> I am not sure, if that is intended, but "lambda" is also a Maxima function (https://maxima.sourceforge.io/docs/manual/maxima_singlepage.html#index-lambda), if it would be automatically converted to the greek letter "λ", it would be impossible to enter this function.

I do not think so. `zeta` and `gamma` are also functions, but there is no such problem with them. And what about `beta` and `theta` ?

**Comment by daute** [comment]
The issue with lambda should be fixed now. And the display of beta and theta seems okay here. Maybe a issue/bug of the used font?

**Comment by gunterkoenigsmann** [comment]
Closing due to inactivity.
